recipes/vtk/.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+*.bash text=lf
+*.sh text=lf
```

Comment:
I work on a Windows machine, and it kept converting newlines. This made my life a little bit easier.